core/src/main/java/org/bitcoinj/core/Locktime.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright by the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bitcoinj.core;
+
+import org.bitcoinj.base.internal.TimeUtils;
+
+import java.time.Instant;
+import java.util.Objects;
+
+/**
+ * <p>
+ * Wrapper for transaction lock time, specified either as a block height or as a timestamp (in seconds
+ * since epoch). Both are encoded into the same long "raw value", as used in the Bitcoin protocol.
+ * The lock time is said to be "not set" if its raw value is zero.
+ * </p>
+ * <p>
+ * Instances of this class are immutable.
+ * </p>
+ */
+public class Locktime {
```

Comment:
Indeed I wantd to put this up for discussion. We sometimes call it Locktime and sometimes LockTime. I think this would be a good opportunity to establish this as a noun?

It also has the advantage of a getter not clashing with its deprecated counterpart (-:

Reply:
You're probably right and `LockTime` is the better name. However how should we rename the new `Transaction.getLocktime()` accessor? `getLockTime()` is already taken. `getWrappedLockTime()`?

Reply:
`.lockTime()`?

Reply:
You're genius! Why didn't I think of it. Will rename.